static/app/components/eventOrGroupHeader.tsx

```diff
@@ -48,17 +45,18 @@ function EventOrGroupHeader({
   hideIcons,
   hideLevel,
   eventId,
-  size = 'normal',
   grouping = false,
   source,
 }: EventOrGroupHeaderProps) {
   const location = useLocation();
 
+  const hasIssuePriority = organization.features.includes('issue-priority-ui');
+
   function getTitleChildren() {
     const {level, isBookmarked, hasSeen} = data as Group;
     return (
       <Fragment>
-        {!hideLevel && level && <GroupLevel level={level} />}
+        {!hideLevel && level && !hasIssuePriority && <GroupLevel level={level} />}
```

Comment:
This hides the old log level indicator. We're moving it to `<EventMessage />` which is already used around the app (and displays the level when passed)